include/htmlrenderer.h

```diff
@@ -109,12 +109,12 @@ class htmlrenderer {
 	bool line_is_nonempty(const std::string& line);
 	unsigned int add_link(std::vector<linkpair>& links,
 		const std::string& link,
-		link_type type);
-	std::string quote_for_stfl(std::string str);
+		LinkType type);
+	std::string quote_for_Stfl(std::string str);
```

Comment:
shouldn't it stay `quote_for_stfl`? capitalizing just one last word seems weird to me.